sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/DocumentSentiment.java

```diff
@@ -13,20 +13,20 @@
 @Immutable
 public final class DocumentSentiment {
     private final SentimentLabel sentiment;
-    private final SentimentScorePerLabel sentimentScores;
+    private final SentimentConfidenceScorePerLabel confidenceScores;
     private final List<SentenceSentiment> sentences;
 
     /**
      * Creates a {@code DocumentSentiment} model that describes the sentiment of the document.
      *
      * @param sentiment the sentiment label of the document
-     * @param sentimentScores the score of sentiment label of the document
+     * @param confidenceScores the score of sentiment label of the document
```

Comment:
the confidence score....
Also, Add to docstrings that the score represents the Softmax score.

Reply:
One sentiment label could have more than one score. So it should be plural? 
e.g., 
DoucentSentimentLabel is negative with 0.1 positive, 0.1 neutral, 0.8 negative

Reply:
One document (or sentence) has one sentiment label and has 3 confidence scores (positive, negative, neutral).
What I am pointing out here, same as Krista, is if you could modify the docstring.
You could use something like https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/textanalytics/Azure.AI.TextAnalytics/src/DocumentSentiment.cs#L33